src/services/report_service.js

```diff
@@ -62,4 +74,10 @@ export default (easypostClient) =>
 
       return this._retrieve(url);
     }
+
+    static _getReportType(reportType) {
```

Comment:
issue: This should have a docstring defining what it does. Regex can get messy so we should have a readable description of it

Also, I thought we had this functionality used elsewhere too for the normal function, do we need to redefine this here?

Reply:
The docstring is added. I don't see we have this helper function in the code.

Reply:
Well done